src/main/io/ledstrip.c

```diff
@@ -105,7 +105,7 @@ static uint8_t previousProfileColorIndex = COLOR_UNDEFINED;
 // Decay the estimated max task duration by 1/(1 << LED_EXEC_TIME_SHIFT) on every invocation
 #define LED_EXEC_TIME_SHIFT             7
 
-#define PROFILE_COLOR_UPDATE_INTERVAL_US 1e6  // normally updates when color changes but this is a 1 second forced update
+#define COLOR_UPDATE_INTERVAL_US 1e6  // normally updates when color changes but this is a 1 second forced update
```

Comment:
1e6 is `double` constant, with possible unintended consequences
```suggestion
#define COLOR_UPDATE_INTERVAL_US ((unsigned)1e6)  // normally updates when color changes but this is a 1 second forced update
```

Reply:
thanks